### PR TITLE
Add the release type of 'Azure Verified'.

### PR DESCRIPTION
**Pull request by LGDoor**
This release type specify a stable and verified version of Jenkins to install, which prevents from deployment failures.

### CODE REVIEW

File: jenkins/install_jenkins.sh [hunk 1]
```diff
@@ -198,8 +198,21 @@ sudo apt-get update --yes
 sudo apt-get install openjdk-8-jre openjdk-8-jre-headless openjdk-8-jdk --yes
 
 #install jenkins
-sudo apt-get install jenkins --yes
-sudo apt-get install jenkins --yes # sometime the first apt-get install jenkins command fails, so we try it twice
+if [[ ${jenkins_release_type} == 'verified' ]]; then
+  jenkins_version=$(curl --silent "${artifacts_location}/jenkins/jenkins-verified-ver")
```
Comment (chenkennt) [inline jenkins/install_jenkins.sh]: ${artifacts_location} contains a tag number, that means jenkins-verified-ver is fixed once you release the template. Then it is no different than you directly hardcode the version number in the script.

Comment (LGDoor) [reply jenkins/install_jenkins.sh]: changed ${artifacts_location} to the master branch in both solution templates and quick start templates.

Comment (clguiman) [reply jenkins/install_jenkins.sh]: That's not a good idea. The odds of someone accidentally breaking master are very high

**Comment by LGDoor** [comment]
Added an additional param for the url of Jenkins version.

**Review by chenkennt — APPROVED** [review]

**Pull request merged**